views/movies/edit-movie.hbs

```diff
@@ -0,0 +1,25 @@
+<h2>Edit movie</h2>
+<form action='/movies/{{movie._id}}/edit' method='post'>
+	<label>
+		Title:
+		<input type='text' name='title' value='{{movie.title}}' />
+	</label>
+	<label>
+		Genre:
+		<input type='text' name='genre' value='{{movie.genre}}' />
+	</label>
+	<label>
+		Plot:
+		<input type='text' name='plot' value='{{movie.plot}}' />
+	</label>
+	<label>
+		Cast:
+
+		<select name='cast'>
```

Comment:
Here I would use the `multiple`, so the user could select multiple celebreties for the movie